src/outbox.erl

```diff
@@ -0,0 +1,125 @@
+%%%-------------------------------------------------------------------
+%%% @doc
+%%%  Module that initializes outboxes and offers their functionalities
+%%% @end
+%%%-------------------------------------------------------------------
+-module(outbox).
+-author("pawel").
+
+%% API
+-export([
+%%    start_outboxes/0,
+    specs_for_supervisor/0,
+    init_outbox/1,
+    post/2,
+    is_empty/1,
+    all_empty/0
+]).
+
+%%%---------------------------
+%% Exported functions
+%%%---------------------------
+
+-spec specs_for_supervisor() -> list(map()).
+specs_for_supervisor() ->
+    lists:map(
+        fun(Zone) -> #{
+            id => list_to_atom("outbox_" ++ binary_to_list(Zone)),
```

Comment:
`child_id()` może mieć dowolny typ. Z reguły jest to atom, ale atomy nie są uwzględniane przez garbage collector, dlatego co do zasady zamierzam się czepiać widząc funkcję `list_to_atom` i podobne. Niech to będzie po prostu binarka.

Reply:
Ok